layers/meta-balena-raspberrypi/recipes-bsp/rpi-eeprom/files/default-config.txt

```diff
@@ -1,4 +1,7 @@
+[all]
 BOOT_UART=0
 WAKE_ON_GPIO=1
 POWER_OFF_ON_HALT=0
 FREEZE_VERSION=0
+HDMI_DELAY=0
+BOOT_ORDER=0xf25641
```

Comment:
@floion just checking, looking at this [internal thread ](https://balena.zulipchat.com/#narrow/stream/360838-balena-io.2Fos.2Fdevices/topic/rpi.20cm4.20v1.2E1) I'm wondering if 0xf25416 would fit more use-cases?

Reply:
I agree it's probably better - let's do it in a different PR as it needs some specific testing.